includes/class-common.php

```diff
@@ -40,6 +42,7 @@ function get_book_markup( $publication ) {
 	<div class="publication book">
 		<h3 class="h5 publication-title font-italic"><?php echo $publication->post_title; ?></h3>
 		<p class="publication-authors"><?php echo $author_string; ?></p>
+		<p class="publication-contributors"><?php echo $contributor_string; ?></p>
```

Comment:
Ideally an `! empty` check should be added before printing this `<p>` since the `publication_contributors` field is optional.